frontend/src/metabase/redux/app.js

```diff
@@ -72,7 +72,30 @@ const isNavbarOpen = handleActions(
   checkIsSidebarInitiallyOpen(),
 );
 
+export const SET_COLLECTION_ID = "metabase/app/SET_COLLECTION_ID";
+export const CLEAR_BREADCRUMBS = "metabase/app/CLEAR_BREADCRUMBS";
+export const setCollectionId = createAction(SET_COLLECTION_ID);
+export const clearBreadcrumbs = createAction(CLEAR_BREADCRUMBS);
+export const getBreadcrumbCollectionId = state =>
```

Comment:
Other selectors for `app` are here `metabase/selectors.app.ts`. Should we move these new ones too?

Reply:
Whoops! I just noticed "getIsNavbarOpen" in this file so I followed that pattern haha